v03_pipeline/lib/tasks/reference_data/update_variant_annotations_table_with_updated_reference_dataset_test.py

```diff
@@ -45,7 +45,7 @@
         hl.tstruct(
             locus=hl.tlocus('GRCh38'),
             alleles=hl.tarray(hl.tstr),
-            PHRED=hl.tint32,
+            PHRED=hl.tfloat32,
```

Comment:
we had a few types in our tests/mock data that weren't right and doing this work caught them 🤩 !